docs/buildah-config.md

```diff
@@ -180,7 +180,7 @@ If names are used, the container should include entries for those names in its
 
 **--volume** *volume*
 
-Add a location in the directory tree which should be marked as a *volume* in any images which will be built using the specified container. Can be used multiple times.
+Add a location in the directory tree which should be marked as a *volume* in any images which will be built using the specified container. Can be used multiple times. If volume has trailing `-`, then the volume is removed from the config
```

Comment:
Please change to `If *volume* has a trailing` ...   and then add a period to the end.

Reply:
Did this comment get missed?

Reply:
Oops, sorry! Just fixed it :)